docs/content/templates/functions.md

```diff
@@ -214,6 +214,23 @@ e.g.
     </ul>
 
 
+### union
+Given two arrays (or slices) A and B, this function will return a new array that contains the elements or objects that belong to either A or to B or to both. The elements supported are strings, integers and floats (only float64).
```

Comment:
This func was motivated by the OR/AND discussion re. the `where` func, but that would only work if if it also support `Pages` (or more general: Slices of structs and pointers).

I guess this func may be useful as is, but it was not what I envisioned.